cypress/src/actions/index.js

```diff
@@ -66,6 +76,13 @@ export const signUpUser = (sign_up, isValid = true) => {
   } else {
     cy.get(fields.authFormUserPassword).eq(1).clear().type(sign_up.shortPassword);
   }
-
+  cy.get('.dropin-picker__select').select('Male');
+  // cy.contains('Male').click();
+  cy.wait(2000);
+  cy
+    .viewport('iphone-x')
+    .percySnapshot('Auth Create Account', { width: 375 })
+    .viewport(1280, 1024)
+    .percySnapshot('Auth Create Account', { width: 1280 });
```

Comment:
Nice, maybe we should wrap this into a util function like`function takeSnapshot(reason:string, widths: number[])`?

I see you repeating this block a bit.